network-api/networkapi/buyersguide/templates/category_nav_links.html

```diff
@@ -2,7 +2,7 @@
 
 {% for cat in sorted_categories %}
 {% if cat.hidden == False %}
-{% if cat.published_product_count > 0 %}
+{% if cat.published_product_page_count > 0 %}
```

Comment:
looking at this, the better solution here is to create a copy of this file as `network-api/networkapi/buyersguide/templates/buyersguid/category_nav_links.html ` (so, with that new app namespace) and then pointing to the new file rather than the original.

Reply:
Agreed. Done!